addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/handler/FreeboxThingHandler.java

```diff
@@ -278,8 +286,9 @@ private void updateCall(CallEntry call, String channelGroup) {
                     new StringType(call.getNumber()));
             updateState(new ChannelUID(getThing().getUID(), channelGroup, CALLDURATION),
                     new DecimalType(call.getDuration()));
-            updateState(new ChannelUID(getThing().getUID(), channelGroup, CALLTIMESTAMP),
```

Comment:
What is the story behind this change?

Reply:
The constructor with Calendar was deprecated in the ESH frameworkj.